### ISSUE TITLE
Broken?

### ISSUE THREAD

**Issue by ryanshrott** [nderkach/airbnb-python#28]
I can’t get this to work anymore. Anyone still use it successfully?

**Comment by purijs** [comment]
Nope, I always get `HTTPError: 400 Client Error: Bad Request for url`

**Comment by Ismar11** [comment]
Some endpoint parameters require updating its values but it's still working. PRs and merge are restricted here, I keep it updated on my [fork](https://github.com/Ismar11/airbnb-python).

**Comment by r3v3r3seEntropy** [comment]
@Ismar11  i have been trying to use your fork, but i get verification error any work around?
